### ISSUE TITLE
suppressing a powershell warning in the context of JEA

### ISSUE THREAD

**Issue by K0nne** [Icinga/icinga-powershell-framework#228]
Hello,

we are using the JEA-feature of powershell for our windows- and hyperv-monitoring. With this we are able to monitor otherwise unreachable metrics like services an windows update with an unpriviged user. The only downside we faced so far is the following powershell-warning, which pollutes the plugin output as well as the service history. The checks itself are running fine.

## Current Behavior
The following warning is shown at every JEA-enabled check execution. It doesn't appear during local executions.

`WARNING: The 'Microsoft.PowerShell.Utility' module was not imported because the 'Microsoft.PowerShell.Utility' snap-in was already imported.`

**plugin output**
![29-03-_2021_11-18-57](https://user-images.githubusercontent.com/34264690/112823159-363fc400-9089-11eb-906a-51b91f543f8e.png)

**service history**
![29-03-_2021_12-00-48](https://user-images.githubusercontent.com/34264690/112823153-3344d380-9089-11eb-92ec-15a7a2fcd686.png)

## Expected Behavior
Is there a way to suppress this warning?

## Possible Solution
I found existing JEA / Powershell issues on that matter:
https://github.com/PowerShell/PowerShell/issues/14650
(see my comment in the linked issue)

## Steps to Reproduce (for bugs)
I tested with Invoke-IcingaCheckCPU
1. configure JEA on the windows host
2. add the needed icinga functions to VisibleFunction in the rolecapabilities-file
3. add the following aliases to the rolecapabilities-file: 
`AliasDefinitions = @{ Name = 'Select-Object' ; Value = 'Microsoft.PowerShell.Utility\Select-Object' }, @{ Name = 'add-member' ; Value = 'Microsoft.PowerShell.Utility\add-member' }, @{ Name = 'new-object' ; Value = 'Microsoft.PowerShell.Utility\new-object' }`
Without this you will face powershell warnings about missing commandlet parameters during check execution, such as:
`Select-Object: A parameter cannot be found that matches parameter name 'ExpandProperty'.`
4. run the check in a JEA-configured session

## Your Environment

* PowerShell Version used (`$PSVersionTable.PSVersion`):
```
Name                           Value
----                           -----
PSVersion                      5.1.14393.3866
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.3866
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

* Operating System and version (`Get-IcingaWindowsInformation Win32_OperatingSystem | Select-Object Version, BuildNumber, Caption`):
```
Version    BuildNumber Caption
-------    ----------- -------
10.0.14393 14393       Microsoft Windows Server 2016 Datacenter
```

**Comment by LordHepipud** [comment]
I added a linked PR which should resolve this issue by clearing the entire host console in case the Framework is initialised with

```powershell
Use-Icinga -Minimal
```

As the new function `Exit-IcingaExecutePlugin` should deal with all exceptions and print them anyway, there is no need to any other kind of message.

Could you please test if this resolves the issue?

**Comment by K0nne** [comment]
Hello,

I have tested the PR. Unfortunately it does not help to suppress the warning, but it helped me to find out the true source of the warning. The addition of `Clear-Host` produced an internal powershell parser error. That message revealed, the culpit to be JEA. Because of this I added a [WarningPreference ](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/new-psrolecapabilityfile?view=powershell-7.1) to our JEA config. 

After the new config was registered, the warning is finally gone.
